src/Compiler/Checking/CheckExpressions.fs

```diff
@@ -8829,10 +8851,12 @@ and TcILFieldItemThen cenv overallTy env finfo tpenv mItem delayed =
 
     | _ ->
         // Get static IL field
-        let expr =
+        let (expr, tpenv), isSpecial =
```

Comment:
What does "special" mean here please?

Reply:
To be frank, I don't know. I copied this behavior from `TcValueItemThen`, where `MakeApplicableExprNoFlex` or `MakeApplicableExprWithFlex` is used based on that value.